### DISCUSSION TITLE
Badges

### DISCUSSION THREAD

**Discussion by astianmuchui** [community/community#20241] [Pull Requests] [answered]
How do I update my pull shark badge? Because I have completed the requirement a lot and just one is showing

**Comment by radar-blip** [comment, marked as answer]
You need to have 16 pull requests merged before it will update to bronze. 
For future reference:
https://github.com/Schweinepriester/github-profile-achievements

**Reply by gunvantpanchal** [reply]
Hi~!
Pull Shark seem to be check your PR record and show Level so you have to merge more pull request
x1 - over 2 PR
x2 - over 16 PR
x3 - over 128 PR
x4 - over 1024 PR

Also "inaccessible" seem to be private PR.
I agree new users might be confused.

**Comment by isinicolle** [comment]
Pull Shark x2	Bronze 🥉	Pull Shark Bronze Badge	16 pull requests merged
Pull Shark x3	Silver 🥈	Pull Shark Silver Badge	128 pull requests merged
Pull Shark x4	Gold 🥇	Pull Shark Gold Badge	1024 pull requests merged

**Comment by blckclov3r** [comment]
I hope this could help you  https://github.com/Schweinepriester/github-profile-achievements
